Tasks/UniversalPackagesV0/package.json

```diff
@@ -17,8 +17,14 @@
     },
     "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
     "dependencies": {
-        "azure-pipelines-task-lib": "2.8.0",
-        "packaging-common": "file:../../_build/Tasks/Common/packaging-common-1.0.1.tgz",
-        "utility-common": "file:../../_build/Tasks/Common/utility-common-1.0.2.tgz"
+        "@types/mocha": "^5.2.7",
+        "@types/node": "^10.17.0",
+        "@types/uuid": "^8.3.0",
```

Comment:
Please move types to `devDependencies`

Reply:
Tried it, but got an error when running the `build` command:
https://github.com/microsoft/azure-pipelines-tasks/blob/3be2d1b2b336cadd5308230cb6496e2e33a3327a/make-util.js#L165-L174

Reply:
Historically, we didn't have `devDependencies` in azure pipelines tasks at all.
Later, the only `typescript` dependency was taken out from `dependencies` to `devDependencies`.